sql/moz-fx-data-shared-prod/telemetry_derived/mobile_engagement_clients_v1/query.sql

```diff
@@ -0,0 +1,151 @@
+WITH mobile_clients_last_seen AS (
```

Comment:
Should we have engagement_client for each namespace separately instead of one "main" table?

Reply:
The only thing is if we do that, I'd definitely want to specify exactly the columns in the unioned view so that if we ever add extra cols to some of the underlying tables to some and not others, the view won't break

Reply:
Having the separate ones and UNIONing at the end was what I was originally expecting, but that said, the logic for engagement tables seems much simpler than what we have for example for retention, so I don't think it's a problem to go directly to the mega table either.
One benefit of having them separate is that if we decide to change something for some app it'll be a smaller backfill, but these tables are not as huge as desktop anyway.